modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,20 +14,41 @@ Provide the following info for each issue if possible:
+** Previously, in the {SMProductShortName} control plane (SMCP), a Grafana data source was not able to set Basic authentication password automatically and users were not able to view metrics from Prometheus in Grafana mesh dashboards. Now, a Grafana data source password is configured under the `secureJsonData` field. Metrics are displayed correctly in dashboards.
+
+* https://issues.redhat.com/browse/OSSM-6148[OSSM-6148] Previously, the {SMPlugin} did not respond when the user clicked on any option in the menu of any node on the **Traffic Graph** page. Now, the plugin responds to the selected option in the menu by redirecting to the corresponding details page.
+
+* https://issues.redhat.com/browse/OSSM-6099[OSSM-6099] Previously, the {SMPlugin} failed to load correctly in an IPv6 cluster. Now, the OSSMC plugin configuration has been modified to ensure proper loading in an IPv6 cluster.
+
+* https://issues.redhat.com/browse/OSSM-5960[OSSM-5960] Previously, the {SMPlugin} did not display notification messages such as backend errors or Istio validations. Now, these notifications are displayed correctly at the top of the plugin page.
+
+* https://issues.redhat.com/browse/OSSM-5959[OSSM-5959] Previously, the {SMPlugin} did not display TLS and Istio certification information in the **Overview** page. Now, this information is displayed correctly.
+
+* https://issues.redhat.com/browse/OSSM-5902[OSSM-5902] Previously, the {SMPlugin} redirected to a "Not Found Page" error when the user clicked on the **Istio config** health symbol on the **Overview** page. Now, the plugin redirects to the correct **Istio config** details page.
+
+* https://issues.redhat.com/browse/OSSM-5541[OSSM-5541] Previously, an Istio operator pod might keep waiting for the leader lease in some restart conditions. Now, the leader election implementation has been enhanced to avoid this issue.
 
 The following issues have been resolved in previous releases:
 
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
-* https://issues.redhat.com/browse/OSSM-3647[OSSM-3647] Previously, in the {SMProductShortName} control plane (SMCP) v2.2 (Istio 1.12), WasmPlugins were applied only to inbound listeners. Since SMCP v2.3 (Istio 1.14), WasmPlugins have been applied to inbound and outbound listeners by default, which introduced regression for users of the 3scale WasmPlugin. Now, the environment variable `APPLY_WASM_PLUGINS_TO_INBOUND_ONLY` is added, which allows safe migration from SMCP v2.2 to v2.3 and v2.4. 
+* https://issues.redhat.com/browse/OSSM-1397[OSSM-1397] Previously, if you removed the `maistra.io/member-of` label from a namespace, the {SMProductShortName} Operator did not automatically reapply the label to the namespace. As a result, sidecar injection did not work in the namespace.
++
+The Operator would reapply the label to the namespace when you made changes to the `ServiceMeshMember` object, which triggered the reconciliation of this member object.
```

Comment:
Simpler verb tenses are always desired per ISG if the meaning is unchanged.

"The Operator would reapply" -> "The Operator reapplied"

Reply:
[OSSM-3647] is from a previous release and is outside the scope of this PR. This may addressed in the future as part of a larger OSSM 2.x doc cleanup effort.